source/js/buyers-guide/components/creep-vote/creep-vote.jsx

```diff
@@ -138,6 +122,14 @@ export default class CreepVote extends React.Component {
    * @returns {jsx} What users see when they haven't voted on this product yet.
    */
   renderVoteAsk() {
+    let unlikelyClasses = classNames("unlikely-glyph btn btn-secondary", {
```

Comment:
Do we need these? Would targeting these elements using CSS's `:checked` selector work? (See existing [implementation](https://github.com/mozilla/foundation.mozilla.org/blob/master/source/js/buyers-guide/components/creep-vote/creep-vote.scss#L45))

Reply:
Yeah, it's necessary. I added `background-color` and `color` on `:checked` but needed the additional class for glyph.